shared/cfg/guest-os/Linux/RHEL.cfg

```diff
@@ -5,6 +5,9 @@
         modprobe_module = acpiphp
     block_hotplug:
         modprobe_module = acpiphp
+    unattended_install:
```

Comment:
should like '    unattended_install, check_block_size.4096_512, check_block_size.512_512, svirt_install, with_installation:', those cases are all for installation, not only unattended_install.